databricks/koalas/namespace.py

```diff
@@ -1674,6 +1701,71 @@ def concat(objs, axis=0, join="outer", ignore_index=False):
                 "and ks.DataFrame are valid".format(name=type(objs).__name__)
             )
 
+    axis = validate_axis(axis)
+    if axis == 1:
+        if isinstance(objs[0], ks.Series):
+            concat_kdf = objs[0].to_frame()
+        else:
+            concat_kdf = objs[0]
+
+        with ks.option_context("compute.ops_on_diff_frames", True):
+
+            def assign_columns(kdf, this_column_labels, that_column_labels):
+                # Note that here intentionally uses `zip_longest` that combine
+                # all columns.
+                for this_label, that_label in itertools.zip_longest(
+                    this_column_labels, that_column_labels
+                ):
+                    yield (kdf._kser_for(this_label), this_label)
+                    yield (kdf._kser_for(that_label), that_label)
+
+            for kser_or_kdf in objs[1:]:
+                if isinstance(kser_or_kdf, Series):
+                    # TODO: there is a corner case to optimize - when the series are from
+                    #   the same DataFrame.
```

Comment:
too many whitespaces?

Reply:
It makes a todo as a sentence :-). 
![Screen Shot 2020-03-18 at 7 02 05 PM](https://user-images.githubusercontent.com/6477701/76948850-04aafd80-694b-11ea-96ae-5e46ad900f50.png)
![Screen Shot 2020-03-18 at 7 02 19 PM](https://user-images.githubusercontent.com/6477701/76948858-070d5780-694b-11ea-8f80-b20479a3951d.png)